ts_scripts/push_nightly.sh

```diff
@@ -0,0 +1,16 @@
+set -ex
+
+rm -r dist || true
+
+
+python setup.py --override-name torchserve-nightly bdist_wheel
+
+if [ -z "$PYPI_TOKEN" ]; then
+    echo "must specify PYPI_TOKEN"
+    exit 1
+fi
+
+python3 -m twine upload \
+    --username __token__ \
+    --password "$PYPI_TOKEN" \
```

Comment:
Can you ensure that the "$PYPI_TOKEN" is not printed in the logs?

Reply:
Just inspect visually and it's not

Reply:
GitHub (and CircleCI for that matter) logging facilities ensure that secrets are always filtered out before being added to the log